plugins/image-prioritizer/readme.txt

```diff
@@ -0,0 +1,58 @@
+=== Image Prioritizer ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.4
+Tested up to:      6.5
+Requires PHP:      7.2
+Stable tag:        0.1.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, images
+
+Optimizes the loading of the LCP image by leveraging client-side detection with real user metrics.
+
+== Description ==
+
+Optimizes the loading of the LCP image by leveraging client-side detection with real user metrics. Currently, this involves adding `fetchpriority=high` to the LCP image and adding preload links for the LCP `img` as well as any LCP `background-image`.
+
+This plugin requires the [Optimization Detective](https://wordpress.org/plugins/optimization-detective/) plugin as a dependency.
+
+TODO: Flesh out description.
```

Comment:
Let's not forget this before release

Reply:
Indeed! I've fleshed it out in the sub-PR: https://github.com/WordPress/performance/pull/1261